common/config/environment.go

```diff
@@ -275,6 +278,10 @@ func (e *CommonEnvironment) Site() string {
 	return e.AgentConfig.Get(DDAgentSite)
 }
 
+func (e *CommonEnvironment) AgentExtraEnvVars() string {
+	return e.AgentConfig.Get(DDAgentExtraEnvVars)
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏

Could you parse the string in this function and return a `map[string]string` so it's easier to re-use it somewhere else in the code?

Reply:
[dc91e32](https://github.com/DataDog/test-infra-definitions/pull/801/commits/dc91e32be9526f573dbc8434b0956bb665377486)